src/main/java/com/microsoft/sqlserver/jdbc/SQLServerConnection.java

```diff
@@ -4416,9 +4463,12 @@ else if (serverMajorVersion >= 9) // Yukon (9.0) --> TDS 7.2 // Prelogin disconn
         if (federatedAuthenticationInfoRequested || federatedAuthenticationRequested) {
             len2 = len2 + writeFedAuthFeatureRequest(false, tdsWriter, fedAuthFeatureExtensionData);
         }
-
-        len2 = len2 + 1; // add 1 to length becaue of FeatureEx terminator
-
+        //Data Classification is always enabled (by default)
+        
+        len2 += writeDataClassificationFeatureRequest(false, tdsWriter);
```

Comment:
hmm why len2? (there is no len1), perhaps a more meaningful name?

Reply:
keeping it the same way as before, not changing existing variables unless required.